lighthouse-cli/bin.ts

```diff
@@ -281,6 +281,14 @@ function runLighthouse(url: string,
       return results;
     })
     .then((results: Results) => {
+      if (url !== results.url) {
```

Comment:
> Come to think of it, thoughts on injecting the warning into the report itself?

#1512 kind of captures the need for this. For stuff that's logged anyway, one easy fix is to capture any warning or error log messages and somehow display them in the report.